test/godeps.list

```diff
@@ -1,6 +1,7 @@
 github.com/gorilla/websocket
 github.com/juju/persistent-cookiejar
 github.com/lxc/lxd/client
+github.com/lxc/lxd/lxd/metrics
```

Comment:
Nope, shouldn't be in there, the test actually did its job at flagging a problem :)